src/Routes/Routes.js

```diff
@@ -40,6 +40,15 @@ const Routes = [
     exact: true,
     component: TaskContent,
   },
+  {
+    path: '/jupyterlab',
+    exact: true,
+    component: JupyterLab,
+  },
+  {
+    path: '/jupyterlab/:path+',
+    component: JupyterLab,
+  },
```

Comment:
@SchafferCPqD @cedric-matheus @LuanEdCosta 
fiquei em dúvida se dá pra fazer a mesma coisa com 1 route apenas; Vocês podem me ajudar, por favor? :)

Reply:
Acredito que o que você está precisando ali seja um parâmetro opcional da rota. Aí quando o parâmetro não for enviado na URL a variável que o componente recebe fica `undefined`

```JSX
<Route path="/to/page/:pathParam?" component={MyPage} />
```

Dá uma olhada: https://stackoverflow.com/questions/35604617/react-router-with-optional-path-parameter

Reply:
@LuanEdCosta obrigado pelo link :)
No final das contas, funcionou fazendo assim: `'/jupyterlab/:path*'`

A rota tinha que dar match nestes 2 casos:
`/jupyterlab` (aqui o ? resolve)
`/jupyterlab/tree/tasks/.../?reset&open=Experiment.ipynb,Deployment.ipynb` (aqui precisava de um + ou ? pra pegar as coisas após uma `/`)

Aparentemente `*?` é reduntante, então só `*` deu certo. :)